src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -206,12 +208,8 @@ def ramp_fit_data(ramp_data, buffsize, save_opt, readnoise_2d, gain_2d,
         raise ValueError("Some of the DQ flags required for ramp_fitting are None.")
 
     if algorithm.upper() == "GLS":
-        # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-        # !!!!! Reference to ReadModel and GainModel changed to simple ndarrays !!!!!
-        # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-        # new_model, int_model, gls_opt_model = gls_fit.gls_ramp_fit(
-        #     model, buffsize, save_opt, readnoise_model, gain_model, max_cores)
-        image_info, integ_info, gls_opt_model = None, None, None
+        image_info, integ_info, gls_opt_info = gls_fit.gls_ramp_fit(
```

Comment:
Since GLS is now an option, update the function description to reflect this.

Reply:
I updated the function description.